index.html

```diff
@@ -20,10 +20,16 @@
   <header>
     <div class="container">
       <div class="h-block">
-        <a class="h2-style">Portfolio</a>
+        <a class="h2-style pff-500">Portfolio</a>
       </div>
       <!--navbar section-->
       <nav class="h-block container fd-rr mr">
+        <ul class="container nav-desktop hidden-el">
+          <li class="li-desktop pff-500">Portfolio</li>
+          <li class="li-desktop pff-500">About</li>
+          <li class="li-desktop pff-500">Contact</li>
+          <li class="li-desktop pff-500"><img class="e-mail-icon-style" src="img/mail-icon.svg" alt="e-mail icon" ></li>
+        </ul>
```

Comment:
- [ ] Please make sure that this menu is being shown in your desktop version. Right now, I can see the hamburger menu for both mobile and desktop versions, so kindly fix this issue.

Reply:
Mobile Version from the inspector
![image](https://user-images.githubusercontent.com/98621809/154823012-66e78b39-e6f9-45ba-96a8-6825e0ca4d45.png)

Desktop version from the inspector
![image](https://user-images.githubusercontent.com/98621809/154823037-ca28adff-2689-427b-8048-931e4d171ad5.png)